neurom/features/morphology.py

```diff
@@ -45,27 +45,44 @@
 
 import warnings
 
+from itertools import chain
 from functools import partial
+from collections.abc import Iterable
 import math
 import numpy as np
 
 from neurom import morphmath
-from neurom.core.morphology import iter_neurites, iter_segments, Morphology
+from neurom.core.morphology import (
+    iter_neurites, iter_sections, iter_segments, iter_points, Morphology
```

Comment:
black doesn't leave a trailing comma?

Reply:
The module is not formatted with black in general. I was thinking of making a pass on the entire package when v4 is merged so that we don't have a lot of irrelevant changes here.

Reply:
before v4 is released, yeah?

Reply:
I don't have a preference. We can do it before the release, yeah.